### ISSUE TITLE
[BUG] "Unknown Import Symbol" for Multi-Subpackage Imports in Namespace Packages

### ISSUE THREAD

**Issue by liblaf** [microsoft/pyright#10056]
### Describe the bug

When importing multiple subpackages (`pkg_a` and `pkg_b`) from a namespace package `org` in a single `from ... import` statement, Pyright incorrectly reports an "unknown import symbol" error for `pkg_a`. This error does not occur when the subpackages are imported individually or in separate `import` statements.

Steps to reproduce:
1. Install `org.pkg_a` as a regular package (e.g., from PyPI).
2. Install `org.pkg_b` as an editable package from a local directory structured as a namespace package.
3. Import both subpackages in a single line: `from org import pkg_a, pkg_b`.
4. Run Pyright, which will report an error for `pkg_a`.

Expected behavior: Both `pkg_a` and `pkg_b` should be recognized as valid imports without errors when imported together in a single statement.

### Code or Screenshots

Directory structure for packages:
```
# org.pkg_a (installed package)
site-packages/
  org/
    pkg_a/
      __init__.py

# org.pkg_b (editable local package)
src/
  org/
    pkg_b/
      __init__.py
```

Code triggering the error:
```python
from org import pkg_a, pkg_b  # Error: "pkg_a" is unknown import symbol
```

Note: The following imports work without errors:
```python
from org import pkg_a
from org import pkg_b
```

### VS Code extension or command-line

I am running **the command-line tool**.

```console
> pyright --version
pyright 1.1.396
```

### Additional Information

- Real-world example reproducing the issue:
  - `org.pkg_a` = [`liblaf.grapes`](https://pypi.org/project/liblaf-grapes) (installed via PyPI)
  - `org.pkg_b` = [`liblaf.melon`](https://github.com/liblaf/melon) (installed in editable mode from local clone)
- Both packages use the `liblaf` namespace.
- Error occurs specifically when combining imports in one statement:
  ```python
  from liblaf import grapes, melon  # "melon" marked as unknown symbol
  ```

### Related Issues

- [#9165](https://github.com/microsoft/pyright/issues/9165) - Almost the same issue. But I think this is a bug that should be fixed (if feasible), because the python interpreter works properly, but `pyright` doesn't work.
